ansible_collections/arista/cvp/docs/modules/cv_validate_v3.rst.md

```diff
@@ -0,0 +1,98 @@
+# cv_validate_v3
+
+Online/Offline configlet validation
+
+Module added in version 3.7.0
+
+<div class="contents" local="" depth="2">
+
+</div>
+
+## Synopsis
+
+CloudVision Portal Validate module to Validate configlets against a
+device on CVP
+
+## Module-specific Options
+
+The following options may be specified for this module:
+
+<table border=1 cellpadding=4>
+
+<tr>
+<th class="head">parameter</th>
+<th class="head">type</th>
+<th class="head">required</th>
+<th class="head">default</th>
+<th class="head">choices</th>
+<th class="head">comments</th>
+</tr>
+
+<tr>
+<td>device<br/><div style="font-size: small;"></div></td>
+<td>list</td>
+<td>yes</td>
+<td></td>
+<td></td>
+<td>
+    <div>CVP device information</div>
+</td>
+</tr>
+
+<tr>
+<td>validate_mode<br/><div style="font-size: small;"></div></td>
+<td>str</td>
+<td>yes</td>
+<td></td>
+<td><ul><li>stop_on_error</li><li>stop_on_warning</li><li>ignore</li></ul></td>
+<td>
+    <div>validation mode</div>
+</td>
+</tr>
+
+</table>
+</br>
+
+## Examples:
+
+    # offline validation
+    - name: offline configlet validation
+      hosts: cv_server
+      connection: local
+      gather_facts: no
+      vars:
+        CVP_DEVICES:
+          - device_name: leaf1
+            search_type: serialNumber #[hostname | serialNumber | fqdn]
+            local_configlets:
+              valid: "interface Ethernet1\n  description test_validate"
+              error: "ruter bgp 1111\n   neighbor 1.1.1.1 remote-bs 111"
+
+      tasks:
+        - name: validate module
+          arista.cvp.cv_validate_v3:
+            device: "{{CVP_DEVICES}}"
+            validate_mode: stop_on_error # | stop_on_warning | valid
+
+    # online validation
+    - name: Online configlet validation
+      hosts: cv_server
+      connection: local
+      gather_facts: no
+      vars:
+        CVP_DEVICES:
+          - device_name: leaf1
+            search_type: fqdn #[hostname | serialNumber | fqdn]
```

Comment:
Wouldn't leaf1 be hostname rather then fqdn? 

```suggestion
            search_type: hostname #[hostname | serialNumber | fqdn]
```

Reply:
I changed the device_name to look like `fqdn`. Tbh, this was an ATD example and in ATD, fqdn, hostname and serialnumber all match!